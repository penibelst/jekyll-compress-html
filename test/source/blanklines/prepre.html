---
layout: compress
---

<div>

TEXT
<pre></pre>

<pre></pre>

</div>
