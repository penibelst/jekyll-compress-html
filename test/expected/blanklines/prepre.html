<div>
TEXT
<pre></pre>
<pre></pre>
</div>
